index.html

```diff
@@ -112,6 +112,13 @@
               status:   "WD",
               publisher:  "W3C Verifiable Credentials Working Group"
             },
+            "JOSE-REGISTRIES": {
+              title:    "The JSON Object Signing and Encryption (JOSE) Registries",
+              href:     "https://www.iana.org/assignments/jose/jose.xhtml",
```

Comment:
```suggestion
              href:     "https://www.iana.org/assignments/jose",
```

Reply:
Possible this will change in the future, better to not ask for XML by file extension